### PR TITLE
new rlglk site-sensor

### PR DESCRIPTION
**Pull request by rumachan**
As we have been discussing :smile_cat:

### CODE REVIEW

File: network/stations.csv
Comment (elidana) [inline network/stations.csv:1723]: since this site is co-located with `GLKZ`, I suggest to use the same `Name`
can you please change `Raoul Green Lake` to `Green Lake` ?

GLKZ entry for this is:
`GLKZ,KI,Green Lake,-29.26068,-177.918038,30,,WGS84,2009-05-02T00:00:00Z,9999-01-01T00:00:00Z`

File: install/dataloggers.csv
Comment (elidana) [inline install/dataloggers.csv:23]: since this site is co-located with GLKZ, I suggest to use the same Name
can you please change `Raoul Green Lake` to `Raoul Island Green Lake` ?

GLKZ entry for this is:
`Kinemetrics,Q330S/6,5440,Raoul Island Green Lake,,2021-10-24T21:28:00Z,9999-01-01T00:00:00Z`

File: install/connections.csv
Comment (elidana) [inline install/connections.csv:744]: since this site is co-located with GLKZ, I suggest to use the same Name
can you please change `Raoul Green Lake` to `Raoul Island Green Lake` ?

GLKZ entry for this is:
`GLKZ,10,Raoul Island Green Lake,,0,2009-05-19T00:00:00Z,9999-01-01T00:00:00Z`

File: network/sites.csv
Comment (elidana) [inline network/sites.csv:1755]: @rumachan , for the coordinates:
the undeployed water and temperature sensors at the sites have these as coordinates:
```
GLKZ,40,-29.26247,-177.91978,10,,WGS84,External GPS Device,2009-05-19T00:00:00Z,2023-01-30T20:17:32Z
GLKZ,80,-29.2624,-177.91978,10,,WGS84,External GPS Device,2009-05-02T00:00:00Z,2023-01-30T20:17:32Z
GLKZ,81,-29.26231,-177.92025,10,,WGS84,External GPS Device,2009-05-02T00:00:00Z,2023-01-30T20:17:32Z
```

do you know if RLGLK sensors have been placed in a different position now? (same comment valid for the coordinates in stations.csv)

**Review by elidana — COMMENTED** [review]
Hi @rumachan , this looks good to me, thanks!!

I have made some comments on the station name and coordinates.

Keen to know what @ozym think about the station name one, and for the coordinates I'm happy with whatever you think is best

**Comment by ozym** [comment]
The PR itself looks fine, I'll leave it up to @rumachan to comment further. I can see either answer for code/locality/name being okay. The goal should probably be related to the user and what is expected (findable). As it is now part of the EN network it really should have a different code - same issues related to stations with multiple networks.

**Comment by elidana** [comment]
thanks @ozym ! I was only commenting on the location name, site code will be `RLGLK` , so a different one (unless I'm missing something from your comment?)

**Comment by rumachan** [comment]
Added "Island" to station name and made site coordinate for 04 same as GLKZ 40,80

**Review by elidana — DISMISSED** [review]
looks good, thanks @rumachan !

**Comment by elidana** [comment]
maybe was not a good idea? the test is failing
```
--- FAIL: TestDataloggers (0.00s)
    --- FAIL: TestDataloggers/check_for_datalogger_installation_place_overlaps (0.00s)
        dataloggers_test.go:36: datalogger CR6:[17445] at Raoul Island Green Lake          has place overlap between 2023-01-30T02:59:59Z and 9999-01-01T00:00:00Z
```

**Comment by rumachan** [comment]
Help @elidana or @ozym I made the requested changes in GitHub but failing Travis checks as it think I have datalogger installed at two places `dataloggers_test.go:36: datalogger CR6:[17445] at Raoul Island Green Lake          has place overlap between 2023-01-30T02:59:59Z and 9999-01-01T00:00:00Z` I presume it thinks `Raoul Green Lake` and `Raoul Island Green Lake`. I don't know how to sort this out as the file only have one entry. Can you help?

**Review by elidana — APPROVED** [review]

**Comment by elidana** [comment]
Let's use `Raoul Green Lake` for the envirosensor for now. this one looks good now and is passing

**Comment by elidana** [comment]
@CBurton90 , can you please have a quick look as well before we merge?

**Comment by ozym** [comment]
@elidana the "Place" in the datalogger table is only used to link this table with the connections table, it's not an official label but needs to be unique (i.e. we can use the site location code (10,20) to differentiate between sensors, but there isn't the same mechanism for dataloggers when there are more than one at a site).

**Comment by rumachan** [comment]
So what you are saying @ozym is that we can't link dataloggers with station code? Meaning we have to stick with what we have? 

In this case the two loggers are, I believe, physically in the same box, so having the same station name is better if possible. Meaning it's clearer to our users.

**Comment by ozym** [comment]
Sorry, may have got lost in translation.

Assume you have a station XXXX where you install 2 dataloggers  A and B.

Also at the site you install 2 sensors X and Y.   The system needs to know which sensor is connected to which datalogger. This is independent of names or data and is contained in the "collennections" file.  i.e. dataloger at site XXXX (called Place but has also a Role, i.e. Primary, and the second one has a Place and a Role, e.g. Secondary) then they are unique and we can join the sensors to the dataloggers.

So you could use the same Place, for the datalogger, but it will require different Roles (this isn't the case for Recorders as they are self contained and don't use the connections file).

**Comment by ozym** [comment]
What is used in the connections (i.e. Place and Role) is just used to match what is in the datalogger table, and is not expected to be used for any external information or meta data. (could just be random values).

**Pull request merged**
